api/src/schemas.ts

```diff
@@ -168,6 +168,36 @@ export const schemas = {
       })
     }
   },
+  updateMyPortfolio: {
+    body: Type.Object({
+      portfolio: Type.Array(
+        Type.Object({
+          description: Type.String(),
+          id: Type.String(),
+          image: Type.String(),
+          title: Type.String(),
+          url: Type.String()
+        })
+      )
```

Comment:
in the old api, it doesn't fail if any of the data is missing, it only fails if the user query fails. 

you can have an empty portfolio item https://www.freecodecamp.org/Sboonny

Reply:
Thanks, Muhammed, I'll update it!